src/index.ts

```diff
@@ -78,27 +82,56 @@ async function logGist(opts: Opts): Promise<void> {
 
 async function runPerformanceTest(opts: Opts): Promise<void> {
     log(`${chalk.gray('[init]')} storing performance results in ${chalk.green(Constants.PERF_FILE)}`);
+
     fs.mkdirSync(path.dirname(Constants.PERF_FILE), { recursive: true });
+    if (fs.existsSync(Constants.PERF_FILE)) {
+        fs.truncateSync(Constants.PERF_FILE);
+    }
+
+    let build: string;
+    if (opts.runtime === 'web') {
```

Comment:
this can be simplified further, to check if runtime is web and there is a githubToken, only then pass build url. Otherwise, options will be same for both runtimes.